css-contain-3/Overview.bs

```diff
@@ -1044,8 +120,6 @@ Changes from <a href="https://www.w3.org/TR/css-contain-2/">CSS Containment Leve
 	* Defines the terms, properties, units, and at-rule
 		needed for [=Container Queries=]
 
-ISSUE: [[css-contain-2#changes]]
-
 <h2 class="no-num" id="acknowledgments">Acknowledgments</h2>
```

Comment:
I have mixed feelings about retaining the Acknowledgments section here.  It seems maybe OK for now, except might make less sense once the draft is repurposed for something else.

Reply:
I was wondering about that too, and for now erred on the side of not deleting acknowledgements, as doing that can be seen as crass if not fully justified. As I did include them in the css-conditional-5 spec, maybe that's enough to remove it from css-contain-3 already?